### ISSUE TITLE
[SF][A11y][ui5-panel]The Panel accordion arrow is missing tooltip

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5509]
### **Bug Description**
Hi team,
The Panel accordion arrow is missing tooltip when hover the arrow icon.

### **Expected Behavior**
The Panel accordion arrow should show tooltip when hover the arrow icon. Just like [openUI5](https://sdk.openui5.org/entity/sap.m.Panel/sample/sap.m.sample.PanelExpanded)
<img width="994" alt="Screen Shot 2022-07-14 at 17 07 48" src="https://user-images.githubusercontent.com/78520316/178946664-e23f38f0-db8c-4b06-8e2e-b40fc1e70120.png">

### **Steps to Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/Panel/'
2. Hover a panel accordion arrow.
3. You can see there is no tooltip.

### **Context**
- UI5 Web Components version: {1.4.0}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SAP SuccessFactors}
- Business impact: {...}

**Comment by TeodorTaushanov** [comment]
Hello @SAP/ui5-webcomponents-topic-rl , take a look of this request.

**Comment by elenastoyanovaa** [comment]
Acc expert analysis:

We should add a tooltip for the icon by including a show-tooltip attribute + accessible-name="Expand/Collapse". Additionally  we should remove the additional styles preventing the pointer events.
